src/app/fsat/explore-opportunities/explore-opportunities-form/fan-data-form/fan-data-form.component.ts

```diff
@@ -146,6 +160,39 @@ export class FanDataFormComponent implements OnInit {
     }
   }
 
+  checkEfficiency(val: number, num: number) {
```

Comment:
This function is unneeded. See comment above. Warnings checked in warnings service.